### DISCUSSION TITLE
v0.0.2

### DISCUSSION THREAD

**Discussion by sanketsarang** [blobcity/autoai#147] [Announcements]
## Key Changes
Includes important bug fixes. Wider model catalogue added. Code generation introduced for both `py` and `ipynb` files. 

## What's Changed
* Update Scaling and Feature Transformation list by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/1
* Auto Data Clean,Feature Selection & YAML Generator by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/2
* fixed issue in identifying problem type by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/4
* Auto Model Selection and Trained model Class by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/6
* Added  setup,pyproject and contributing.md update by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/7
* setup config update by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/11
* minor fixes by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/15
* minor fix by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/16
* minor value change by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/17
* Removed access to other functions by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/19
* Added Cv Score log output by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/22
* Added Metric Statsics by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/23
* Pandas DataFrame support to train function by @balamurugan1603 in https://github.com/blobcity/autoai/pull/27
* solves issue #20 by @sreyan-ghosh in https://github.com/blobcity/autoai/pull/26
* minor fix by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/29
* minor fix by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/30
* changed parameter from file_path to file by @sanketsarang in https://github.com/blobcity/autoai/pull/36
* Added XGBClassifier by @balamurugan1603 in https://github.com/blobcity/autoai/pull/50
* Loading CSV from URL by @balamurugan1603 in https://github.com/blobcity/autoai/pull/35
* XGBClassifier Parameter Config fix by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/56
* BernoulliNB classifier config hyperparams updated by @melan96 in https://github.com/blobcity/autoai/pull/55
* minor fix by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/75
* minor fix by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/76
* Minor fix by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/79
* Added example Regression & Classification Tests by @sanketsarang in https://github.com/blobcity/autoai/pull/80
* Load Functionality Change by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/81
* Moved test files to base folder by @sanketsarang in https://github.com/blobcity/autoai/pull/84
* adaboost-clf added. hyperparams adjusted. by @melan96 in https://github.com/blobcity/autoai/pull/77
* regressor-poissonregressor added to source by @melan96 in https://github.com/blobcity/autoai/pull/83
* Added HistGradientBoostingClassifier to Classifier Config by @Devolta05 in https://github.com/blobcity/autoai/pull/85
* minor fix by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/87
* regressor sgd fixes added on lossfunction by @melan96 in https://github.com/blobcity/autoai/pull/86
* Added the parameters for the Nearest Centroid by @Tanuj2552 in https://github.com/blobcity/autoai/pull/88
* minor fix by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/90
* minor fix by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/92
* Added SGDClassifier to `classifier_config.py` by @Devolta05 in https://github.com/blobcity/autoai/pull/91
* minor fix by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/93
* Enhanced Docs by @sanketsarang in https://github.com/blobcity/autoai/pull/94
* Added AdaBoostRegressor by @26tanishabanik in https://github.com/blobcity/autoai/pull/89
* minor fix by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/96
* Configuration Minor fix by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/97
* minor fix by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/101
* minor fix by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/102
* Major Enhancement by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/105
* Added the parameters for Lasso Regressor by @Tanuj2552 in https://github.com/blobcity/autoai/pull/98
* Added RadiusNeighborsClassifier by @aadityasinha-dotcom in https://github.com/blobcity/autoai/pull/51
* Modified the parameters for Lasso Regressor by @Tanuj2552 in https://github.com/blobcity/autoai/pull/100
* Added Lars model to regressor_config.py by @SaharshLaud in https://github.com/blobcity/autoai/pull/106
* Minor fix by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/113
* Added Categorical Naive Bayes to `classifier_config,py` by @Devolta05 in https://github.com/blobcity/autoai/pull/99
* Minor fix by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/117
* Minor fix by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/118
* Added LassoLars by @naresh1205 in https://github.com/blobcity/autoai/pull/114
* Added Bayesian Ridge Config by @Bhumika0201 in https://github.com/blobcity/autoai/pull/119
* Minor fix by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/120
* Minor bug fix by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/124
* Added configuration for ElasticNet by @Bhumika0201 in https://github.com/blobcity/autoai/pull/121
* config added for MultinomialNB by @Bhumika0201 in https://github.com/blobcity/autoai/pull/126
* replaced unique( ) function of target_length by @Cipher-unhsiV in https://github.com/blobcity/autoai/pull/104
* Add XGBoost Regressor by @vedantbahel in https://github.com/blobcity/autoai/pull/125
* Added ARDRegressor model to regressor_config.py by @SaharshLaud in https://github.com/blobcity/autoai/pull/127
* CodeGen - Support for ipynb files by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/128
* Added catboost regressor to regressor configuration by @Devolta05 in https://github.com/blobcity/autoai/pull/110
* Minor fix CatboostRegressor configuration by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/130
* Added Gamma Regressor by @aadityasinha-dotcom in https://github.com/blobcity/autoai/pull/129
* Minor addition by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/131
* minor fix by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/132
* Added PassiveAggressiveRegressor by @aadityasinha-dotcom in https://github.com/blobcity/autoai/pull/135
* Added RadiusNeighborRegressor by @aadityasinha-dotcom in https://github.com/blobcity/autoai/pull/134
* Added LightGBM model to regressor_config.py  by @SaharshLaud in https://github.com/blobcity/autoai/pull/133
* Minor fix by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/138
* Added Perceptron classifier to classifier_config.py by @SaharshLaud in https://github.com/blobcity/autoai/pull/142
* minor fix by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/143
* Hacktoberfest Issue-137 Drop rows with more than 50% NANs by @TamannaBhasin27 in https://github.com/blobcity/autoai/pull/144
* Minor Enhancement by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/145
* Minor fix by @Thilakraj1998 in https://github.com/blobcity/autoai/pull/146

## New Contributors
* @Thilakraj1998 made their first contribution in https://github.com/blobcity/autoai/pull/1
* @balamurugan1603 made their first contribution in https://github.com/blobcity/autoai/pull/27
* @sreyan-ghosh made their first contribution in https://github.com/blobcity/autoai/pull/26
* @sanketsarang made their first contribution in https://github.com/blobcity/autoai/pull/36
* @melan96 made their first contribution in https://github.com/blobcity/autoai/pull/55
* @Devolta05 made their first contribution in https://github.com/blobcity/autoai/pull/85
* @Tanuj2552 made their first contribution in https://github.com/blobcity/autoai/pull/88
* @26tanishabanik made their first contribution in https://github.com/blobcity/autoai/pull/89
* @aadityasinha-dotcom made their first contribution in https://github.com/blobcity/autoai/pull/51
* @SaharshLaud made their first contribution in https://github.com/blobcity/autoai/pull/106
* @naresh1205 made their first contribution in https://github.com/blobcity/autoai/pull/114
* @Bhumika0201 made their first contribution in https://github.com/blobcity/autoai/pull/119
* @Cipher-unhsiV made their first contribution in https://github.com/blobcity/autoai/pull/104
* @vedantbahel made their first contribution in https://github.com/blobcity/autoai/pull/125
* @TamannaBhasin27 made their first contribution in https://github.com/blobcity/autoai/pull/144

**Full Changelog**: https://github.com/blobcity/autoai/commits/v0.0.2

<hr /><em>This discussion was created from the release <a href='https://github.com/blobcity/autoai/releases/tag/v0.0.2'>v0.0.2</a>.</em>
